libraries/helper.rb

```diff
@@ -23,4 +23,31 @@ def entity_uuid
       return Chef::DataCollector::Messages.node_uuid
     end
   end
+
+  def get_tests_for_runner
+    profiles = node['audit']['profiles']
+    profiles_array = []
+    profiles.each { |profile| profiles_array.push(profile[:url] || profile[:git] || profile[:source]) }
+
+    profiles.each { |profile|
+      if profile.length == 1 && profile[:name]
+        profiles_array.push("supermarket://" + profile[:name])
```

Comment:
I think we want to give the runner the whole hash directly instead of transforming that to urls